### ISSUE TITLE
Constant alignment

### ISSUE THREAD

**Issue by xepozz** [PHP-CS-Fixer/PHP-CS-Fixer#5162]
## Support question

How I can prevent this correction?
![image](https://user-images.githubusercontent.com/6815714/95231513-e84db000-080b-11eb-8d14-f8d8abb4838d.png)

**Comment by SpacePossum** [comment]
Hi,

Most likely you are running the `binary_operator_spaces` with `['operators' => ['=' => 'align']`. If you don't want that you can either reconfigure the rule or remove/disable it. 
If this doesn't work please share your configuration and which version of the tool you are using so we can help out : )

**Comment by xepozz** [comment]
I've set the next config
```php
'binary_operator_spaces' => [
    'default' => 'align_single_space_minimal',
],
```
and now constants are aligned, but it caused the next side effect
![image](https://user-images.githubusercontent.com/6815714/95254725-fe1d9e00-0828-11eb-8181-2b7992615531.png)

I want to set alignments only for constants scope. PHPStorm and Editorconfig provides this ability.

**Comment by SpacePossum** [comment]
hi,
currently we've no options that apply on (not) constants only, only on operator level

**Comment by xepozz** [comment]
Are you tell me about plans for this feature? Can I help you with it?

**Comment by Wirone** [comment]
@xepozz there are no plans for it. Core team has limited capacity and won't work on such edge cases, but if you want to provide PR with this feature covered, we can review it 🙂.

**Comment by Wirone** [comment]
I believe it was fixed in #7175 🙂.
